app/Models/Sprint.php

```diff
@@ -81,4 +82,16 @@ public function getLastUpdatedDate(): Carbon
     {
         return $this->lastUpdated;
     }
+
+    public function containsToday(): Bool
```

Comment:
I think it needs to be lowercase `bool` vs `Bool`
```suggestion
    public function containsToday(): bool
```

Reply:
Ugh, you're right but nothing seemed to complain like I would have expected! I'll change that.